indico_mattermost_bot.py

```diff
@@ -78,6 +115,8 @@ def read_config(config_file):
         'channels': channels
     }
 
+def is_fetching_past_events(bot):
```

Comment:
I'd underscore the function name, since we're doing the same above (e.g. `_dt(...)`).